chapters/statements.tex

```diff
@@ -430,6 +431,25 @@ \subsubsection{Restrictions on When-Statements}\label{restrictions-on-when-state
 \end{lstlisting}
 \end{example}
 
+\paragraph*{Restrictions on statements within when-statements:}\label{restrictions-on-statements-within-when-statements}
+
+\begin{nonnormative}
+In contrast to when-equations, \cref{restrictions-on-equations-within-when-equations},
+there are no additional restrictions within when-statements:
```

Comment:
(Just noting that this is not using sentence-based (but rather "semantical") line breaking.  Semantical line breaks are generally not as robust as sentence-based ones, and harder to predict, making cross-project search or search/replace operations more likely to miss matches.)

Reply:
Let's consider this resolved by comment https://github.com/modelica/ModelicaSpecification/pull/2718#issuecomment-731154142.